tests/src/rules/no-extraneous-dependencies.js

```diff
@@ -69,14 +69,6 @@ ruleTester.run('no-extraneous-dependencies', rule, {
       code: 'import "doctrine"',
       options: [{packageDir: path.join(__dirname, '../../../')}],
     }),
-    test({
```

Comment:
what was wrong with these tests? do they not still pass given your change? I would think the first one should still be good since it won't resolve?